components/sbm-core/src/main/java/org/springframework/sbm/engine/recipe/RewriteRecipeLoader.java

```diff
@@ -24,7 +24,7 @@
 import java.util.Optional;
 
 //
-// @Component
+@Component
```

Comment:
Do we need this component? If so, could you maybe comment on line 39 so the recipes are not added to the list here?